scripts/executableConvention.fsx

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env -S dotnet fsi
+
+open System
+open System.IO
+
+#r "nuget: Mono.Unix, Version=7.1.0-final.1.21458.1"
+#load "../src/FileConventions/Library.fs"
+
+let NotInDir (dirName: string) (fileInfo: FileInfo) =
+    not(
+        fileInfo.FullName.Contains
+            $"%c{Path.DirectorySeparatorChar}%s{dirName}%c{Path.DirectorySeparatorChar}"
+    )
+
+let invalidFiles =
+    Directory.GetFiles(".", "*.fsx", SearchOption.AllDirectories)
+    |> Seq.map(fun pathStr -> FileInfo pathStr)
+    |> Seq.filter(NotInDir "node_modules")
+    |> Seq.filter(NotInDir ".git")
+    |> Seq.filter(NotInDir "bin")
+    |> Seq.filter(NotInDir "obj")
+    |> Seq.filter(NotInDir "DummyFiles")
```

Comment:
@realmarv I believe there's a new func in FileConventions that has these list above already, let's use it here

Reply:
Done